### DISCUSSION TITLE
Anyone ever seen oddly garbled field names before?

### DISCUSSION THREAD

**Discussion by tronboto** [vectordotdev/vector#21747] [Q&A] [answered]
Hey. So I've just noticed that some of our logs which are coming in through the `logstash` source (it might not be exclusive to this source but it's definitely not a majority) have garbled fields in elasticsearch. For example:

```
dtn_request_ocessing_start_time
dtn_request_processing__finish_time
dtn_request_processing_fih_time
dtn_request_processing_finh_time
dtn_request_processing_finis_time
dtn_request_processing_finish__time
dtn_request_processing_finish_h_time
dtn_request_processing_finish_time
dtn_request_processing_finishish_time
dtn_request_processing_finisish_time
dtn_request_processing_finisnish_time
dtn_request_processing_finistime
dtn_request_processing_fish_time
dtn_request_processing_start_time
```

Looking back, this only seems to have started happening when we migrated from logstash to vector. I'm not too sure where to even start so would appreciate if anyone has any ideas.

There are some error messages for the source but it's not very clear to me what they mean (we don't have acknowledgements enabled on any of our sinks):

![image](https://github.com/user-attachments/assets/f39594c8-0cd5-44fe-ba35-54d2a5847729)

**Comment by pront** [comment]
Hi @tronboto, I haven't seen anything like this before. Do you have any more details to share like source configuration and setup?

**Comment by tronboto** [comment, marked as answer]
Nothing to see here @pront . Sorry for any time you spent thinking about this - these fields were coming from the user's filebeat instance.
